docs/metrics.md

```diff
@@ -0,0 +1,54 @@
+# Watchdog Metrics
+*Last Update: 2018-04-20*
+
+## Analysis
+Questions we want to answer with metrics data include:
+
+- Overall throughput performance:
+  - Consumer submission to submission to PhotoDNA (time in queue)
+  - Response from PhotoDNA (time waiting for reply)
+  - Response to consumer API (time to reply)
+  - The sum of the above to give an easy health measure
+- Throughput data for positive identifications since they require
+  manual intervention:
+  - Number of positively flagged images
+    - Breakdown of images not yet reviewed and under review
+  - Number of images confirmed vs falsely identified
+- The number of items in the message queue
+- Total number of images processed
+  - Breakdown of positive vs negative responses
+
+Each of these should be available globally, as well as broken down per consumer
+application.
+
+
+## Collection
+This project uses Ping Centre to collect metrics data.  Pings will be sent as
+JSON blobs.  All pings will include the following fields:
+- Current timestamp: string
```

Comment:
is this the only shared metric across the pings?

Reply:
it's the only one I could think of